README.md

```diff
@@ -55,6 +62,12 @@ where:
 
 The validator can then be executed via bash commands. See the [preceeding instructions for command line usage](#run-the-app-via-command-line).
 
+### Visualize the results
+In the output directory, the following files will be created:
+- `report.html`: the validation report in HTML format. It can be opened in a browser.
+- `report.json`: the validation report in JSON format.
+- `system_errors.json`: this file will be created every-time the validator is run. If no system errors were encountered, this file will be empty.
+
```

Comment:
I wonder if it's worth just linking up to the previous section where you document the output files, instead of repeating it verbatim here?  It seems like this will just get out of sync over time.